packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-expandable-answer.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  buildSmartSnippet,
+  SmartSnippet,
+  SmartSnippetState,
+} from '@coveo/headless';
+import {h, Component, State, Prop, Element, Listen} from '@stencil/core';
+import ArrowDown from '../../images/arrow-down.svg';
+import {
+  InitializeBindings,
+  Bindings,
+  BindStateToController,
+} from '../../utils/initialization-utils';
+
+/**
+ * @internal
+ */
+@Component({
```

Comment:
I decided to make this a full stencil component because it needs render the snippet first in order to calculate its height, then re-render in order to display or remove the "show more/less" button.